js/backbone.upload-manager.js

```diff
@@ -69,6 +69,13 @@
             // Add local events handlers
             this.bindLocal();
         },
+        // Max Wong: Cannot add same file after cancelled #21
+        _clearPreviousAddFileInput: function() {
+        	var self = this;
```

Comment:
Why is that relevant to us a `self` variable here when you can directly use `this` ?